11_bank_card.py

```diff
@@ -0,0 +1,124 @@
+"""
+
+Hometask 14 lambda.
+
+# write class for bank card.
+# Class should reflect card lifecycle, card operations etc.
+# use CVV, PIN, Name, Surname , end date, card_num as initial params
+# class should have in addition to common logic some class attributes,
+# as minimum one classmethod and
+# as minimum  one staticmethod, two or more getters/setters,
+# __str_ magic method
+# do not forget about block
+
+"""
+
+import datetime
+import random
+
+
+class BankCard:
+    """Make class."""
+
+    bank = 'Best Bank'
+    card = 'VISA'
+
+    def __init__(self, name, surname, balance):
+        """Make init."""
+        self.name = name
+        self.surname = surname
+        self.balance = balance
+        self.card_num = self.card_number
```

Comment:
card_pin, exp_date, card_cvv  all these attributes should be with type protected.
You need to organize  setters and getters  for these attributes. 
card_num could be private and have only getter.
when you generate card_num  - function should return full card number
You need to hide fist 12 digits only within card __str__ method logic